__tests__/app.test.js

```diff
@@ -250,4 +250,57 @@ describe("News Express App", () => {
         });
     });
   });
+  describe.only("POST /api/articles/:article_id/comments", () => {
+    test("201:  object of the posted", () => {
+      const comment = { icellusedkars: "Wagon Wheels" };
+
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send(comment)
+        .expect(201)
+        .then(({ body }) => {
+          expect(body).toEqual({ body: "Wagon Wheels" });
+        });
+    });
+    test("404: when an article id is requested that does not exist, responds with a 404 error and an appropriate message", () => {
+      const comment = { icellusedkars: "Wagon Wheels" };
+
+      return request(app)
+        .post("/api/articles/188/comments")
+        .send(comment)
+        .expect(404)
+        .then(({ body }) => {
+          expect(body).toEqual({
+            msg: "No article found with article ID: 188",
+          });
+        });
+    });
+    test("404: when comment is posted with author whose username is not in the users table, responds with an appropriate error message ", () => {
+      const comment = { questioningmyexistance: "Wagon Wheels" };
+
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send(comment)
+        .expect(404)
+        .then(({ body }) => {
+          expect(body).toEqual({
+            msg: `No user found with username: questioningmyexistance`,
+          });
+        });
+    });
+
+    test("400: if invalid id of invalid type is requested, responds with appropriate message ", () => {
+      const comment = { questioningmyexistance: "Wagon Wheels" };
+
+      return request(app)
+        .post("/api/articles/potato/comments")
+        .send(comment)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body).toEqual({
+            msg: `invalid input type`,
+          });
+        });
+    });
+  });
```

Comment:
The post body should have a key of username and body like {username: "icellusedcars", body: "Never gonna give you up"} Also I would add tests for if the post has missing info like if the username or body is missing